app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,22 @@
 class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+
+    if @comment.save
+      redirect_to user_posts_path(current_user)
+    else
+      render :new
+    end
```

Comment:
- [OPTIONAL] Same here, It would be better to redirect the user to the post/comments page after a successful post creation. eg: `redirect_to post_path(@comment.post)`

Reply:
Thanks. Updated